django/db/backends/sqlite3/base.py

```diff
@@ -76,6 +76,10 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         'TimeField': 'time',
         'UUIDField': 'char(32)',
     }
+    data_type_check_constraints = {
+        'PositiveIntegerField': '"%(column)s" >= 0',
+        'PositiveSmallIntegerField': '"%(column)s" >= 0',
+    }
```

Comment:
I guess this is required because `unsigned` isn't enforced by SQLite at all?

Reply:
Correct